src/Host/Client/Impl/Session/RSessionProvider.cs

```diff
@@ -138,13 +142,15 @@ public class RSessionProvider : IRSessionProvider {
                         foreach (var session in sessions) {
                             session.CancelSwitchingBroker();
                         }
+                        BrokerChangeFailed?.Invoke(this, EventArgs.Empty);
```

Comment:
Ok

---
In reply to: [79260010](https://github.com/Microsoft/RTVS/pull/2341#discussion_r79260010) [](ancestors = 79260010)